website/content/en/docs/cli/operator-sdk_run_local.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "operator-sdk run local"
+---
+## operator-sdk run local
+
+Run an Operator locally
+
+### Synopsis
+
+This command will run your Operator locally by building the operator binary
+with the ability to access a kubernetes cluster using a kubeconfig file
+
+```
+operator-sdk run local [flags]
+```
+
+### Options
+
+```
+      --enable-delve             Start the operator using the delve debugger
+      --go-ldflags string        Set Go linker options
+  -h, --help                     help for local
+      --kubeconfig string        The file path to kubernetes configuration file. Defaults to location specified by $KUBECONFIG, or to default file rules if not set
+      --operator-flags string    The flags that the operator needs. Example: "--flag1 value1 --flag2=value2"
+      --watch-namespace string   The namespace where the operator watches for changes. Set "" for AllNamespaces, set "ns1,ns2" for MultiNamespace
```

Comment:
Should we not make clear that the watch-namespace is not valid for the new layout?

Reply:
See https://github.com/operator-framework/operator-sdk/pull/3067/files#r427374500. This description should be changed to be specific to `run local` though, since the current one is geared towards OLM.